lib/Backend/Lower.cpp

```diff
@@ -6825,8 +6824,7 @@ Lowerer::GenerateStFldWithCachedType(IR::Instr *instrStFld, bool* continueAsHelp
 
     if (hasTypeCheckBailout)
     {
-        AssertMsg(PHASE_ON1(Js::ObjTypeSpecIsolatedFldOpsWithBailOutPhase) || !propertySymOpnd->IsTypeDead(),
-            "Why does a field store have a type check bailout, if its type is dead?");
+        AssertMsg(PHASE_ON1(Js::ObjTypeSpecIsolatedFldOpsWithBailOutPhase) || !propertySymOpnd->IsTypeDead(), "Why does a field load have a type check bailout, if its type is dead?");
```

Comment:
Correction -- the one above says "load" and this one originally said "store" but now says "load" -- was that change intentional?

---
In reply to: [79080126](https://github.com/Microsoft/ChakraCore/pull/1578#discussion_r79080126) [](ancestors = 79080126)